ClientCRUD.cs

```diff
@@ -37,11 +38,30 @@ public Client Get(string Id)
             }
             return client;
         }
+        //
+        public bool CheckUniqueDni(string dni)
```

Comment:
Relacionado a la sugerencia https://github.com/csampedro/Proyecto-Agencia_de_Alquiler_de_Autos/pull/7/files#r707798349:

```suggestion
        private bool CheckUniqueDni(string dni)
```